kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkManager.java

```diff
@@ -101,12 +102,16 @@ private static ExecutorService queryThreadPool() {
     // CompletableFuture.allOf and converting the CompletableFuture<Void> to
     // CompletableFuture<List<SearchResult>>
     CompletableFuture<List<SearchResult<T>>> searchResults = CompletableFutures.allAsList(queries);
```

Comment:
Is CompletableFutures abstraction still needed after this change?

Reply:
frixed